apps/etterlatte-behandling/src/main/kotlin/behandling/migrering/MigreringService.kt

```diff
@@ -64,7 +65,11 @@ class MigreringService(
                 )
                 val nyopprettaOppgave =
                     oppgaveService.hentOppgaverForSak(it.sak.id).first { o -> o.referanse == it.id.toString() }
-                oppgaveService.tildelSaksbehandler(nyopprettaOppgave.id, pesys)
+                try {
+                    oppgaveService.tildelSaksbehandler(nyopprettaOppgave.id, pesys)
+                } catch (e: OppgavenHarAlleredeSaksbehandlerException) {
+                    logger.info("Oppgaven har allerede saksbehandler. Ikke så farlig for migrering, så vi kjører på", e)
```

Comment:
I hvilke tilfeller inntreffer egentlig dette?

Reply:
Har fått det når migrering har feila, og så prøver eg på nytt med samme sak. Litt usikker på om det skjer når avbryt behandling blir kalt, viss avbryt behandling _ikkje_ blir kalt eller begge delar.

Reply:
Burde vi kanskje sørge for at avbryt behandling blir kalt med en gang det feiler slik at vi ikke risikerer at noen saksbehandlere starter å se på det? Da skal det vel opprettes ny oppgave når vi forsøker igjen, eller? 🤔

Reply:
Bør kanskje grave litt meir i det her og lage ei meir eksplisitt og spesialisert feilhandtering, ja. Skal grave litt i loggane og prøve å finne ut kva det er som (ikkje) skjer.

Reply:
Fann ein meir ordentleg måte å gjera det på: https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2461